lib/adapters/class-adapter.php

```diff
@@ -41,6 +41,13 @@ abstract class Adapter implements Hookable {
 	 */
 	private bool $allow_empty_search = false;
 
+	/**
+	 * Whether to enable phrase matching in queries.
```

Comment:
Is it worth adding the link from the es documentation? So that anyone looking in can better understand what es feature this implements?

Reply:
I can add some links!